### PR TITLE
feat: add respec pipeline

### LINKED ISSUE #2: Setup project automation and tooling
- [x] Issue templates
- [ ] dash-dependencies --> no mvn or gradle nor package-lock files. thus no dash.
- [x] respec pipeline
- [x] empty respec project
- [ ] github pages enabled

requires #1

### PR DESCRIPTION
**Pull request by arnoweiss**
PR will stay in draft mode until a committer has enabled github pages with source "github actions".

closes #2

### CODE REVIEW

File: .github/workflows/autopublish.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Auto-Publish
+on:
+  push:
```
Comment (paullatzelsperger) [inline .github/workflows/autopublish.yaml:3]: IMO it's generally a good idea to add a `workflow_dispatch` trigger too, should we ever need to manually start the job, e.g. from a feature branch, or for some other reason.

**Review by paullatzelsperger — APPROVED** [review]
LGTM, let me know when you are ready to merge.

**Comment by arnoweiss** [comment]
gh pages enabled by https://github.com/eclipse-dataspace-dcp/.eclipsefdn/pull/2

**Comment by arnoweiss** [comment]
> LGTM, let me know when you are ready to merge.

ready

**Pull request merged**